parameterized/parameterized.py

```diff
@@ -386,6 +386,29 @@ def check_input_values(cls, input_values):
             input_values = list(input_values)
         return [ param.from_decorator(p) for p in input_values ]
 
+
+    @classmethod
+    def parameterized_class(cls, properties, test_values):
```

Comment:
Please use `attribute_names` and `input_values` to be consistent with other naming